laboratory-maniac.md

```diff
@@ -68,6 +71,28 @@ Laboratory Maniac in hand. Learning all these piles takes some practice
 but may allow for fast wins, although not cheaper than a normal 
 Tendrils-based wins.
 
+## Winning with Laboratory Maniac Using TW/AoI
+
+The use of TW/AoI allows for some subtle changes in the way you build LM piles:
+```
+GP in hand:
+-> GP, [TW/AoI], LED, GP, LM BBB+[1UU/2R] (6) 6 life
+
+CB in play:
+-> [TW/AoI], LED, GP, GP, LM BBB+[1UU/2R] (6) 6 life
```

Comment:
Apart from the difference in life cost, there is reason to have a different ordering of cards as there is no real difference with 'GP in hand' scenario, right? Or am I missing something?